### PR TITLE
EWS: links in map report and increase timeout in reporting rates

### PR DESCRIPTION
**Pull request by agrzywinski**
last comments in:
http://manage.dimagi.com/default.asp?167209
http://manage.dimagi.com/default.asp?167135

### CODE REVIEW

File: custom/ewsghana/templates/ewsghana/map_template.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        var content = '<div style="margin: 0 0 30px 120px;">';
+
+        $.each(slugs, function() {
+            content += '<a class="btn" style="margin-left: 10px" href="' +
+                    url.substring(0, previousSlash + 1) + this[0] + url.substring(lastSlash) +
+                    '">' + this[1] + '</a>';
+        });
+
+        content += '</div>';
+        $('legend').after(content);
```
Comment (czue) [inline custom/ewsghana/templates/ewsghana/map_template.html]: this looks copy-pasted from here: https://github.com/dimagi/commcare-hq/blob/master/custom/ewsghana/templates/ewsghana/multi_report.html#L334

is it possible to pull it out into a shared template?

**Comment by czue** [comment]
@agrzywinski assume you'll ping me when this is updated or if you disagree with my suggestion

**Comment by agrzywinski** [comment]
@czue I've updated PR

**Comment by czue** [comment]
thanks!

**Pull request merged**